cat-test/pubspec.yaml

```diff
@@ -0,0 +1,5 @@
+name: cat_test
+version: 0.0.1
+dependencies:
+  test: any
```

Comment:
We didn't always, but now we recommend that people use version constraints even for dev dependencies. This helps ensure that other people downloading your package can run its tests successfully.